Makefile

```
@@ -86,7 +86,7 @@ workspaces-vanilla-tf-destroy:
 workspaces-vanilla-porter-build:
 	echo -e "\n\e[34m»»» 🧩 \e[Building vanilla workspace bundle\e[0m..." \
 	&& . ./devops/scripts/check_dependencies.sh porter \
-	&& cd ./workspaces/vanilla/ && porter build
+	&& cd ./workspaces/vanilla/ && porter build --debug
```

I assume the debug flag should not be set for the final bundle?
```suggestion
	&& cd ./workspaces/vanilla/ && porter build
```

---

Hmm, build is done by infra developers, so debug is useful. Can take it off, but I had to keep putting it back on.

---

I'm going to leave these in. The make commands are used for local deployments during the development process - they aren't part of the bundle - it gives more verbose output on the `porter build` command, rather than creating a "debug bundle". Without debug , errors are not much use, porter gives very little output. Same goes for if CD fails, we need to know why.

If we decide later its causing a problem, we can take them off.